components/css/partials/_accordion.scss

```diff
@@ -32,5 +41,29 @@
 				content:"\f0fe";
 			}
 		}
+		@each $color,$pair in $colorPairs-accessible{
+			&-#{$color}{
+				background-color:nth($pair,1);
+				color:nth($pair,2);
+			}
+		}
+	}
+	&_list{
+		margin:0;
+		padding:0 20px;
+		list-style-type:none;
+	}
+	&_item{
+		border-top:solid 1px #000;
+		margin:0;
+		padding:15px 0 !important;
+		&:first-child{
+			border-top:none;
+		}
+		@each $color,$pair in $colorPairs-accessible{
+			&-#{$color}{
+				border-top-color:nth($pair,1);
+			}
+		}
 	}
 }
```

Comment:
Also, I realized that our color pairs doesn't include Garrett's additional grays: 
```
$color-gray90:#2e2e2e;
$color-gray70:#707070;
$color-gray40:#aaaaaa;
$color-gray15:#dedede;
```

Reply:
Good point. Added.